dlc-manager/src/manager.rs

```diff
@@ -1231,6 +1231,117 @@ where
         Ok(())
     }
 
+    fn try_finalize_settled_closing_channel(
+        &self,
+        signed_channel: SignedChannel,
+    ) -> Result<(), Error> {
+        let (settle_tx, &is_offer, &is_initiator) = get_signed_channel_state!(
+            signed_channel,
+            SettledClosing,
+            settle_transaction,
+            is_offer,
+            is_initiator
+        )?;
+
+        if self
+            .blockchain
+            .get_transaction_confirmations(&settle_tx.txid())?
+            >= CET_NSEQUENCE
```

Comment:
The constant `CET_NSEQUENCE` is a bit misleading here as this has nothing to do with CETs. Maybe specifying a separate constant for the settle tx timelock makes it clearer?

Reply:
I agree, but this is how it was specified originally in the settle transaction.